### ISSUE TITLE
allow empty defaults for environment vars in docker-compose.yml

### ISSUE THREAD

**Issue by bisrael** [convox/rack#1338]
There appears to be no way to set a blank default for environment variables in `docker-compose.yml`:

``` yaml
version: "2"
services:
  web:
    environemnt:
    - VAR_1=
    - VAR_2=""
```

`convox start|doctor` will complain that `VAR_1` is missing, but set `VAR_2` not to an empty string as one might imagine, but to a string literally containing two quotation marks (i.e.: `"\"\""`)

i can see two resolutions:
1. Accept `VAR_1=` form to mean blank string
2. Disallow `VAR_1=` form, accept `VAR_2=""` as empty string by dropping the quotes

No. 2 seems more able to handle edge cases including variables whose defaults may contain desired trailing spacing (e.g.: `VAR_3="      "` becomes acceptable to cover that case)

Thanks!

**Comment by MiguelMoll** [comment]
Would empty defaults also carry into deployments or just local? A workaround for `convox start` would be to set  `VAR_1=` in an `.env` file. This is an interesting request though. 👍

**Comment by bisrael** [comment]
i would expect them to carry over into deployments like any other variable set there (assuming that's teh current behavior)

it's valid in every environment i've put into production to have empty env vars

**Comment by bisrael** [comment]
@MiguelMoll

**Comment by MiguelMoll** [comment]
Thanks for the info @bisrael 👍 . Will see how we can get this into our work queue.

**Comment by ddollar** [comment]
Check out #1471 for some examples of how to handle null values in YAML.

**Comment by idris** [comment]
Looks like this was fixed in #1656

**Comment by nzoschke** [comment]
This is fixed. Thanks for the report.
